engine/api/src/main/java/pl/touk/nussknacker/engine/api/validation/LiteralInt.java

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.engine.api.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface LiteralInt {
```

Comment:
I don't like idea to introduce annotation for each specific Validator. It will introduce a lot of boilerplate.
I see such options:
1. Introduce only one annotation `@LiteralValue` and in logic determining validators look at this annoatation + type of parameter
2. In logic determining validator to take a look if someone picked simple string editor + type is int and return `LiteralintValidtor` for such case

Reply:
I've provided `@Literal` adnotation with type arg.

Reply:
Annotation is ok, but `type` parameter inside it is not. You already have type of parameter. Not need to enforce developer to duplicate this information